### PR TITLE
[FranceInter] add thumbnail url

### PR DESCRIPTION
**Pull request by renalid**
Adding thumbnail url

## Please follow the guide below

- You will be asked some questions, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *pull request* (like that [x])
- Use *Preview* tab to see how your *pull request* will actually look like

---

### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [adding new extractor tutorial](https://github.com/ytdl-org/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/ytdl-org/youtube-dl#youtube-dl-coding-conventions) sections
- [x] [Searched](https://github.com/ytdl-org/youtube-dl/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [x] Checked the code with [flake8](https://pypi.python.org/pypi/flake8)

### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [ ] Bug fix
- [x] Improvement
- [ ] New extractor
- [ ] New feature

---

### Description of your *pull request* and other information

I add the thumbnail url getting from the webpage open graph

### CODE REVIEW

File: youtube_dl/extractor/franceinter.py [hunk 1]
```diff
@@ -31,6 +31,7 @@ def _real_extract(self, url):
 
         title = self._og_search_title(webpage)
         description = self._og_search_description(webpage)
+        thumbnail = self._og_search_thumbnail(webpage)
```
Comment (remitamine) [inline youtube_dl/extractor/franceinter.py:34]:
```suggestion
        thumbnail = self._html_search_meta(['og:image', 'twitter:image'], webpage)
```

File: youtube_dl/extractor/franceinter.py [hunk 2]
```diff
@@ -16,6 +16,7 @@ class FranceInterIE(InfoExtractor):
             'ext': 'mp3',
             'title': 'Affaire Cahuzac : le contentieux du compte en Suisse',
             'description': 'md5:401969c5d318c061f86bda1fa359292b',
+            'thumbnail': 'md5:48455373092572b23164123db7598c2e',
```
Comment (remitamine) [inline youtube_dl/extractor/franceinter.py:19]:
```suggestion
            'thumbnail': r're:^https?://.*\.jpg',
```

**Comment by renalid** [comment]
Thanks @remitamine for your help ! :-)

**Comment by remitamine** [comment]
thanks for your contributions.

**Pull request merged**